src/infrastructure/rendering/font_manager.py

```
@@ -30,6 +30,62 @@
+            return await ensure_table_font(_configured_proxy, _configured_timeout)
+        except Exception as ex:  # 防御：兜底未知异常，避免后台任务静默崩溃
+            logger.warning(
+                "表格字体后台预取异常: err_type=%s, err=%s", type(ex).__name__, ex
+            )
+            return None
+
+    _prefetch_task = asyncio.create_task(_run())
+
+
+async def ensure_table_font_runtime() -> Path | None:
+    """渲染路径的按需门控：返回已就绪字体，必要且已配置时才下载。
```

Ruff (0.15.14)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)